spec/lib/message_queue_processor_spec.rb

```diff
@@ -40,6 +40,7 @@
         "content_id" => content_item["content_id"],
         "page_path" => content_item["base_path"],
         "title" => content_item["title"],
+        "public_updated_at" => JSON.parse(Time.zone.parse(content_item["public_updated_at"]).to_json),
```

Comment:
Why `.to_json` and `JSON.parse`?

Reply:
It was the only way I could find to get the timestamp in the right format.

`content_item["public_updated_at"]`:

       expected: {"content_id"=>"39a271f2-45e6-405a-9ac2-b2eae5e8ed47", "page_path"=>"/page-path/1", "public_updated_at"=>"2014-05-06T12:01:00+00:00", "title"=>"Page path 1"}
            got: {"content_id"=>"39a271f2-45e6-405a-9ac2-b2eae5e8ed47", "page_path"=>"/page-path/1", "public_updated_at"=>"2014-05-06T13:01:00.000+01:00", "title"=>"Page path 1"}

`Time.zone.parse(content_item["public_updated_at"]).to_s`:

       expected: {"content_id"=>"e41afcdc-2b39-4544-a78c-30bd58a801cb", "page_path"=>"/page-path/1", "public_updated_at"=>"2014-05-06 13:01:00 +0100", "title"=>"Page path 1"}
            got: {"content_id"=>"e41afcdc-2b39-4544-a78c-30bd58a801cb", "page_path"=>"/page-path/1", "public_updated_at"=>"2014-05-06T13:01:00.000+01:00", "title"=>"Page path 1"}

`Time.zone.parse(content_item["public_updated_at"]).to_json`:

       expected: {"content_id"=>"1d6fb2ae-cc04-4e2a-a7c6-7930cc8249fb", "page_path"=>"/page-path/1", "public_updated_at"=>"\"2014-05-06T13:01:00.000+01:00\"", "title"=>"Page path 1"}
            got: {"content_id"=>"1d6fb2ae-cc04-4e2a-a7c6-7930cc8249fb", "page_path"=>"/page-path/1", "public_updated_at"=>"2014-05-06T13:01:00.000+01:00", "title"=>"Page path 1"}

`JSON.parse(Time.zone.parse(content_item["public_updated_at"]).to_json)`:

works

Reply:
I see... that's annoying